<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startWindow</class>
 <widget class="QMainWindow" name="startWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>550</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>startWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="loginLBL">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>500</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="loginBTN">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Sans Pro</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="usernameGroup">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>440</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>400</width>
       <height>3</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="editUsername">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>400</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLabel" name="isUsernameCorrect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>10</width>
       <height>47</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="passwordGroup">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>440</width>
      <height>75</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="editPassword">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>400</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eyeBTN">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>15</y>
       <width>46</width>
       <height>47</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>354</width>
       <height>3</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="isPassCorrect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>10</width>
       <height>47</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_1">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>400</y>
      <width>260</width>
      <height>60</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>10</y>
       <width>170</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans Pro</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Not a member?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signupBTN">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>5</y>
       <width>70</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 85, 255);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>SignUp</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>5</width>
      <height>525</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="signupLBL">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>0</y>
      <width>495</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Create Account</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="usernameGroup_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>440</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>48</y>
       <width>400</width>
       <height>3</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri Light</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="editUsername_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>8</y>
       <width>400</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLabel" name="isUsernameCorrect_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>10</width>
       <height>38</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>40</y>
      <width>495</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="passwordGroup_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>440</width>
      <height>125</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="Line" name="line_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>48</y>
       <width>354</width>
       <height>3</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri Light</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="editPassword_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>8</y>
       <width>400</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLabel" name="isPasswordCorrect_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>10</width>
       <height>38</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="editPassword_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>400</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Confrim Password:</string>
     </property>
    </widget>
    <widget class="Line" name="line_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>354</width>
       <height>3</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri Light</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="isPasswordCorrect_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>72</y>
       <width>10</width>
       <height>38</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="eyeBTN_2">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>8</y>
       <width>46</width>
       <height>43</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="eyeBTN_3">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>70</y>
       <width>46</width>
       <height>43</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="phoneAndCaptchaBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>260</y>
      <width>440</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QComboBox" name="phoneComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>145</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editPhoneNum">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="showCaptcha">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="refreshBTN">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="captchaLineEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>160</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>enter captcha code...</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="signupBTN_2">
    <property name="geometry">
     <rect>
      <x>675</x>
      <y>440</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Sans Pro</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>SignUp</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
